docs: new aboutguide page (#1419)
asyncapi/website: pages/docs/community/styleguide/aboutguide.md

---
title: About Style Guide

> description: This style guide page guides contributors on how to write documentation.

Reviewer: ```suggestion
description: This document welcomes documentation contributors.
```

weight: 10
---

## Introduction

> Welcome to the AsyncAPI Style Guide. This page guides contributors on how to write, structure, and format content in documentation.

Reviewer: ```suggestion
Welcome to the AsyncAPI Style Guide! This document guides contributors on writing, structuring, and formatting content in documentation.
```

These guidelines help us maintain language, tone, and voice throughout the website. It also makes it easier for multiple contributors to work together on a single doc while ensuring consistency, accuracy, and clarity. 

> ## Writing Style

Reviewer: ```suggestion
## Writing style
```

Below are some recommendations to consider when writing your pages.

### Identify target audience
Before you begin planning and constructing your content, knowing who you are writing for and how the readers will benefit from your content is important.
This will help you in strategizing and simplifying your writing process.

### Research

> Before creating your first draft, it is beneficial to research similar resources and use them as references. Avoid plagiarism or copying and pasting from other websites unless the content already exists on our site.

Reviewer: ```suggestion
Before creating your first draft, it is beneficial to research similar resources and use them as references. Avoid copying and pasting content from other websites or Artificial Intelligence chatbots.
```
